src/component/absensi/shift/Shift.vue

```diff
@@ -19,74 +17,93 @@
                 </tr>
             </thead>
             <tbody>
-                <tr>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-
+                <tr v-for="item in shiftItems"
+                    :key="item.id"
+                    :item="item">
```

Comment:
@aamesaa the `:key="item.id" :item="item">`is still there, does it breaks the whole system when those 2 is not removed?